pilot/pkg/security/model/trace.go

```diff
@@ -1,52 +0,0 @@
-// Copyright 2017 Istio Authors
-//
```

Comment:
Why was tracing config inside security/model? Am I missing something?

Reply:
I was wondering this too. I think it's just a mistake

Reply:
Yeah I think it's a mistake.